### PR TITLE
Dynamic Navigation for List View

### PR DESCRIPTION
**Pull request by bordoni**
_Ref:_ [C#126084](http://central.tri.be/issues/126084)
---
https://github.com/moderntribe/tribe-common/pull/1035

📹 http://i.bordoni.me/0457b851789b

### CODE REVIEW

**Comment by lucatume** [comment]
I've updated cod and tests per https://github.com/moderntribe/the-events-calendar/pull/2535

File: src/Tribe/Views/V2/Hooks.php [hunk 1]
```diff
@@ -136,12 +136,23 @@ public function register_rest_endpoints() {
 			/**
 			 * @todo  Make sure we do proper handling of cache longer then 12h.
 			 */
-			'permission_callback' => function ( \WP_REST_Request $request ) {
+			'permission_callback' => static function ( \WP_REST_Request $request ) {
```
Comment (borkweb) [inline src/Tribe/Views/V2/Hooks.php]: TIL

File: src/resources/js/views/manager.js [hunk 2]
```diff
@@ -323,6 +332,8 @@ tribe.events.views.manager = {};
 
 		var $html = $( data );
 
+		console.log( data );
```
Comment (borkweb) [inline src/resources/js/views/manager.js]: Remove?

File: src/views/v2/events-bar/form.php [hunk 3]
```diff
@@ -14,12 +14,17 @@
  */
 ?>
 <div class="tribe-events-c-events-bar__form">
-	<div class="tribe-common-c-search">
+	<form
+		class="tribe-common-c-search"
+		action=""
```
Comment (borkweb) [inline src/views/v2/events-bar/form.php]: If blank, we don't need to include this, right?

File: tests/views_rest/V2/ContextCest.php [hunk 4]
```diff
@@ -28,10 +28,12 @@ public function should_alter_the_view_context_according_to_rest_request_paramete
 		$I->seeResponseCodeIs( 200 );
 		$I->seeResponseIsJson();
 		$I->seeResponseContainsJson( [
-			'view'      => 'reflector',
-			'url'       => $url,
-			'_wpnonce'  => $nonce,
-			'view_data' => [
+			'eventDisplay'           => 'reflector',
```
Comment (borkweb) [inline tests/views_rest/V2/ContextCest.php]: Spacing needs to be fixed here.

**Pull request merged**